### PR TITLE
remove fe level change link

### PR DESCRIPTION
**Pull request by goodviber**
### Context
When creating a further education course, the check your answers change link leads back to the course level page, an error is raised when filling in the form with Primary or Secondary options.

### Checklist

- [ ] Make sure all information from the Trello card is in here
- [ ] Attach to Trello card
- [ ] Rebased main
- [ ] Cleaned commit history
- [ ] Tested by running locally

### CODE REVIEW

File: app/views/publish/courses/confirmation.html.erb [hunk 1]
```diff
@@ -31,10 +31,14 @@
...
-            visually_hidden_text: "level"
-          ) %>
+          <% if course.is_further_education? %>
+            <% row.action %>
+          <% else %>
+            <% row.action(
+              href: new_publish_provider_recruitment_cycle_courses_level_path(course.provider.provider_code, course.recruitment_cycle.year, params.to_unsafe_h.merge(goto_confirmation: true)),
+              visually_hidden_text: "level"
+            ) %>
+          <% end %>
```
Comment (defong) [inline app/views/publish/courses/confirmation.html.erb:41]: I think its a bad idea to allow for change link at any level

Also can we have tests

**Review by defong — CHANGES_REQUESTED** [review]

File: app/views/publish/courses/confirmation.html.erb (hunk 1 above)
Comment (goodviber) [reply app/views/publish/courses/confirmation.html.erb:41]: To confirm, you think we should remove the change link for all levels of course, FE, Primary, Secondary ?

Comment (defong) [reply app/views/publish/courses/confirmation.html.erb:41]: Yes we should remove the change link for at levels.

There is not enough work done to support switching from level to level as there will be values passed over that is not relevant for the level specified.

**Review by defong — APPROVED** [review]

File: app/views/publish/courses/confirmation.html.erb (hunk 1 above)
Comment (darokel) [reply app/views/publish/courses/confirmation.html.erb]: Could we also remove the change link for the SEND on this page? It'll essentially allow changing the level too

Comment (goodviber) [reply app/views/publish/courses/confirmation.html.erb]: Just saw this, will do

**Review by darokel — APPROVED** [review]

**Pull request merged**